src/pages/signIn/searchPage/components/Molecules/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+import styles from './SearchForm.module.css';
+
+//Future import { useTheme }? (Dark Theme) and styles in the code to be replaced
+
+
+ const SearchForm = ({ onSearch }) => {
+ // const { theme } = useTheme();
+  const [query, setQuery] = useState('');
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    onSearch(query);
+  };
+
+  return (
+    <form
+    className={styles.searchForm}
+      //className={`${styles.searchForm} ${
+       // theme === 'dark' ? styles.darkTheme : ''
+     // }`}
+      onSubmit={handleSubmit}
+    >
+      <input
+       className={styles.input}
+       // className={`${styles.input} ${
+         // theme === 'dark' ? styles.darkTheme : ''
+       // }`}
+        type="text"
+        autoComplete="off"
+        autoFocus
+        required
+        minLength={3}
+        placeholder="Search..."
+        name="query"
+        value={query}
+        onChange={e => setQuery(e.target.value)}
```

Comment:
Create handle and pass as a on change event

Reply:
Done